PowerUp/app/src/main/res/values/strings.xml

```diff
@@ -18,5 +18,11 @@
     <string name="back_to_map">Back to Map</string>
     <string name="replay">Replay</string>
     <string name="pref_previously_started">pref_previously_started</string>
+    <string name="intro_screen_1_text">Welcome to PowerUp! Lets take you through a tour of our application so you can give your best!</string>
+    <string name="intro_screen_2_text">You are Rosie, and on the right is your boyfriend Marco!</string>
+    <string name="intro_screen_3_text">These are different places in your neighborhood. Each place creates a new Scenario.</string>
+    <string name="intro_screen_4_text">Different conversations are created and you can choose from the given answers.</string>
```

Comment:
Same sentences end up with a full stop while others don't. Choose an approach and use the same for all sentences for coherency